app/services/synchronize_source_service.rb

```diff
@@ -0,0 +1,63 @@
+class SynchronizeSourceService
+  def initialize(source)
+    @source = source
+  end
+
+  def execute
+    after = if @source.last_updated_at.nil?
+              nil
+            else
+              @source.last_updated_at.to_s(:db)
+            end
+    @result = interactor.send_request Lit::Source::LOCALIZATIONS_PATH,
+                                      after: after
+    return if @result.nil? || !@result.is_a?(Array)
+    synchronize_localizations
+    update_timestamps
```

Comment:
`synchronize_localizations` is fine because it calls `save!` so that it'll break on failure; however, `update_timestamps` is coded so that any failure will be silent - perhaps we could also make it `save!` instead of `save`.